### PR TITLE
README first draft

### PR DESCRIPTION
**Pull request by flexyford**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,80 @@
...
 
 ## Installation
 
-* `git clone` this repository
-* `npm install`
-* `bower install`
+* `ember install ember-impagination`
 
-## Running
+## Usage
```
Comment (cowboyd) [inline README.md]: In the `Usage` section, I think most people will be using the component, so let's optimize for that case first. Afterwards, you can say "under the covers, the imagination-dataset component is just creating an impagination dataset. You can use it directly if you want. Here's how.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,80 @@
 # Ember-impagination
 [![Build Status](https://travis-ci.org/thefrontside/ember-impagination.svg)](https://travis-ci.org/thefrontside/ember-impagination)
 
-This README outlines the details of collaborating on this Ember addon.
+[Impagination](https://github.com/flexyford/impagination) implementation for Ember.
```
Comment (cowboyd) [inline README.md]: I'd put the link to the demo right up there.

File: README.md [hunk 3]
```diff
@@ -1,26 +1,80 @@
...
+
+## Demo
```
Comment (cowboyd) [inline README.md]: Let's put the explanation of the demo with the demo.

File: README.md (hunk 2 above)
Comment (flexyford) [inline README.md]: yup, just had to makes a gh-pages branch first :smiley:

**Pull request merged**